src/components/ENSListPage/ENSListPage.tsx

```diff
@@ -421,7 +445,7 @@ export default class ENSListPage extends React.PureComponent<Props, State> {
               activePage={page}
               setPage={page => this.setState({ page })}
               totalPages={totalPages}
-              empty={() => null}
+              empty={() => this.renderEmptyEnsList()}
```

Comment:
This might work as well if the typings are not strict:
```suggestion
              empty={this.renderEmptyEnsList}
```

Reply:
The build failed with this 😕